main.py

```
@@ -0,0 +1,198 @@
+import streamlit as st
+import pandas as pd
+import io
+from company_research import CompanyResearcher, get_openai_api_key, create_sample_csv
+
+
+def main():
```

Pylint (3.3.7)

[refactor] Too many local variables (23/15)

(R0914)